src/network/NetworkController.ts

```diff
@@ -288,6 +299,36 @@ export class NetworkController extends BaseController<
     });
     this.refreshNetwork();
   }
+
+  getEIP1559Compatibility() {
+    const { properties = {} } = this.state;
+
+    if (!properties.isEIP1559Compatible) {
+      if (!this.ethQuery || !this.ethQuery.sendAsync) {
```

Comment:
I think we can use optional chaining here, if our project supports it:

```
if (!this?.ethQuery?.sendAsync) {
```

Reply:
resolved :)